www/routes/faq.tsx

```diff
@@ -0,0 +1,92 @@
+export default FAQ;
+export { handler };
+
+import { Handlers, PageProps } from "$fresh/server.ts";
+import { frontMatter, gfm } from "../utils/markdown.ts";
+import { asset, Head } from "$fresh/runtime.ts";
+
+import Footer from "../components/Footer.tsx";
+import Header from "../components/Header.tsx";
+
+interface Data {
+  page: {
+    markdown: string;
+    data: Record<string, unknown>;
+  };
+}
+
+const handler = {
+  async GET(_req, ctx) {
+    const url = new URL(`../../docs/faq.md`, import.meta.url);
+    const fileContent = await Deno.readTextFile(url);
+    const { body, attrs } = frontMatter<Record<string, unknown>>(fileContent);
+
+    const data = {
+      page: {
+        markdown: body,
+        data: attrs ?? {},
+      },
+    } satisfies Data;
+
+    return ctx.render(data);
+  },
+} satisfies Handlers;
```

Comment:
If applied, this suggestion will also require you to remove the `export { handler }` at the top of the file.
```suggestion
export const handler: Handlers<Data> = {
  async GET(_req, ctx) {
    const url = new URL(`../../docs/faq.md`, import.meta.url);
    const fileContent = await Deno.readTextFile(url);
    const { body, attrs } = frontMatter<Record<string, unknown>>(fileContent);

    const data = {
      page: {
        markdown: body,
        data: attrs ?? {},
      },
    } satisfies Data;

    return ctx.render(data);
  },
};
```

Reply:
I'll adjust that.